src/plugins/geomap/demo/config-fr.js

```diff
@@ -281,35 +307,35 @@ var wet_boew_geomap = {
 		},
 		{
 			title: "GeoJSON (CartoDB)",
-			caption: "Ceci est un exemple d'un jeu de données JSON chargé à partir d'un site externe, dans ce cas-ci les caméras de circulation de la ville d'Ottawa à partir du site Carto DB.",
+			caption: "Ceci est un exemple de jeu de données chargé à partir d'une ressource GeoJSON distante, dans ce cas les caméras de circulation dans la ville d'Ottawa à partir de l'API CartoDB",
 			type: "geojson",
 			url: "//stephenott.cartodb.com/api/v2/sql",
 			params: {
 				format: "GeoJSON",
 				q: "SELECT * FROM traffic_cameras LIMIT 25"
 			},
 			attributes: {
-				location_desc: "Emplacement",
+				location_desc: "Location",
 				longitude: "Latitude",
 				latitude: "Longitude",
-				updated_at: "Dernière mise à jour"
+				updated_at: "Last updated"
```

Comment:
Can we keep the French term "Dernière mise à jour"?

Reply:
Fixed